lib/dockerregistry/inventory.go

```diff
@@ -1856,7 +1862,7 @@ func SplitRegistryImagePath(
 }
 
 // nolint[lll]
-func mkPopulateRequestsForPromotionEdges(
+func MKPopulateRequestsForPromotionEdges(
```

Comment:
Please add a comment for this function, now that it is exported.

Reply:
Maybe something like "... returns a PopulateRequests which can generate requests to be processed down the road."